src/platform/apollolake/include/platform/memory.h

```diff
@@ -148,6 +148,7 @@
 #define HEAP_RT_COUNT256		128
 #define HEAP_RT_COUNT512		8
 #define HEAP_RT_COUNT1024		4
+#define HEAP_RT_COUNT4096		2
```

Comment:
Should this PR add also 2048, 8192, and 16384 with number of buffers as zero for now? I recall there's need for up to about 15 kB for new components. It will be then simpler to add the needed amount of storage with single line edit.

Reply:
Allocate this as a buffer today. Ideally we need to drop rballoc and have our allocator select from the best heap.

Reply:
@singalsu @lgirdwood again this was just my quick test/fix to get larger buffers allocated in dsp and test the eq. I'll let someone else to define the best memory configuration.

Reply:
ok, since we need allocator update can we remove rballoc() and have logic in rmalloc that picks the buffer heap for large allocations.